jmclient/jmclient/support.py

```diff
@@ -198,15 +194,30 @@ def weighted_order_choose(orders, n):
     return orders[chosen_order_index]
 
 
+def random_under_max_order_choose(orders, n):
+    # orders are already pre-filtered for max_cj_fee
+    return random.choice(orders)
+
+
 def cheapest_order_choose(orders, n):
     """
     Return the cheapest order from the orders.
     """
     return orders[0]
 
 
+def _get_check_max_fee(max_fee_rel, max_fee_abs, cjvalue):
```

Comment:
I'd prefer a different function name here, like "is_within_max_limits" or so, because a boolean return of True/False actually intuitively makes sense then.